src/app/index.js

```diff
@@ -143,6 +154,7 @@ var Index = (function () {
         document.getElementById('aggregate').addEventListener('change', onAggregationChange);
 
         UIkit.util.on('#metricSelector', 'moved', onMetricMoved);
+        UIkit.util.on('#metricSelector', 'change', onMetricChange);
```

Comment:
Where did you saw the documentation for this event?
When you reorder the metrics have fired both events and saved both configurations.

Reply:
I could not reproduce that.